styles/styles.css

```diff
@@ -92,265 +92,223 @@ h1 {
 
 .primary-btn {
-  background: linear-gradient(
-    45deg,
-    var(--primary-color) 0%,
-    var(--accent-color) 100%
-  );
-  border-radius: 4px;
-  overflow: hidden;
-  cursor: pointer;
+    background: linear-gradient(45deg, var(--primary-color) 0%, var(--accent-color) 100%);
+    border-radius: 4px;
+    overflow: hidden;
+    cursor: pointer;
 }
 
 .secondary-btn {
-  color: var(--principal-text-color);
-  width: 100px;
-  background-color: var(--secondary-color);
-  border-radius: 4px;
-  height: 50px;
-  cursor: pointer;
+    color: var(--principal-text-color);
+    width: 100px;
+    background-color: var(--secondary-color);
+    border-radius: 4px;
+    height: 50px;
+    cursor: pointer;
 }
 
 .tag-btn {
-  background-color: var(--principal-color-v2);
-  color: var(--principal-text-color);
-  padding: 5px 10px;
-  border-radius: 25px;
-  text-decoration: none;
-  font-size: 14px;
+    background-color: var(--principal-color-v2);
+    color: var(--principal-text-color);
+    padding: 5px 10px;
+    border-radius: 25px;
+    text-decoration: none;
+    font-size: 14px;
 }
 
 .tag-btn::before {
-  font-weight: bold;
-  color: var(--accent-color);
-  content: "# ";
+    font-weight: bold;
+    color: var(--accent-color);
+    content: "# ";
 }
 
 .flex-wrap {
-  flex-wrap: wrap;
+    flex-wrap: wrap;
 }
 
 /* Share modal */
 
-.modal {
-  height: 100vh;
-  width: 100%;
-  display: none;
-  position: fixed;
-}
-
-.display-modal {
-  display: flex !important;
-  justify-content: center;
-  align-items: center;
-}
-
-.modal::before {
-  content: "";
-  width: 100%;
-  height: 100vh;
-  position: absolute;
-  background-color: black;
-  opacity: 0.8;
-  top: 0;
-  left: 0;
-}
-
-.share-modal {
-  color: black;
-  border-radius: 2rem;
-  padding: 2rem;
-  width: 250px;
-  z-index: 101;
-  background-color: white;
-  text-align: center;
-}
-
-.share-head {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-}
-
-.share-head i {
-  font-size: 1.3rem;
-  cursor: pointer;
-}
-.social-links {
-  list-style: none;
-  display: flex;
-  gap: 2rem;
-  font-size: 3rem;
-  padding-left: 0;
-}
-
-.social-links li {
-  cursor: pointer;
-}
-
-.copy-btn {
-  width: 100%;
-}
-
-#github-link {
-  position: fixed;
-  top: 0;
-  right: 0;
-  width: 100px;
-  height: 100px;
-  /* transform: translateX(-50%); */
-  clip-path: polygon(1% 0, 100% 100%, 100% 1%);
-  background-color: #2b3137;
-}
-
-#github-link a {
-  width: 100px;
-  height: 100px;
-  text-decoration: none;
-  color: #fff;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  padding-left: 45px;
-  padding-top: 10px;
-  text-align: center;
-  width: 50px;
-  height: 50px;
-}
-
-#github-link a img {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 20px;
-  height: 20px;
-  vertical-align: middle;
-  background-position: center;
-}
-#github-link h1 {
-  font-size: 10px;
-  font-family: sans-serif;
```

Comment:
The deletion of this particular lines will result in breaking the GitHub link feature which was introduced in issue #38.